### DISCUSSION TITLE
Access to Copilot via Github Student Developer Pack

### DISCUSSION THREAD

**Discussion by lirtm** [community/community#108652] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hello. Got access today after verification to the Github Student Developer Pack program.
At the moment I have a 30-day trial of Copilot connected for some time.
I don't have the button "You're eligible to use copilot for free", which is mentioned in the access instructions.

Is this because of the trial?

**Comment by davecheney** [comment, marked as answer]
@lirtm please contact support who can cancel your 30 day trial and switch you to your student-2024 coupon.

**Reply by lirtm** [reply]
Ok, just support.
And, if I may, one more question that support is probably solving as well:

I already have a DigitalOcean account. Now I have a promo to DO with 200$ credit.
But neither on the education mail account nor on my main account linked to GitHub does this promo apply.

Error: The GitHub Student Pack promotion has already been applied to another team. Please switch to the team it was applied to use your credits.

What could be the problem? Thank you!
